docs/guides/chip_tool_guide.md

```diff
@@ -410,10 +414,14 @@ The CHIP Tool can run in one of the following modes:
     [1650992689511] [32397:1415601] CHIP: [TOO] Run command failure: ../../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout
     ```
 
+    Moreover, when using the single-command mode, the CHIP Tool will establish a
+    new CASE session with every command sent (and close the existing one).
+
 -   Interactive mode - In this mode, a command will terminate with an error if
     it does not complete within the timeout period. However, the CHIP Tool will
     not be terminated and it will not terminate processes that previous commands
-    have started.
+    have started. Moreover, when using the interactive mode, the CHIP Tool will
+    establish a CASE session only on the first command sent.
```

Comment:
```suggestion
    establish a new CASE session only on the first command sent. On the following commands invoked, it will use the existing session.
```